### PR TITLE
Feature 2198 zenodo

### LINKED ISSUE #2198: Add .zenodo.json file to add metadata about releases
## Describe the Task ##
Add a [.zenodo.json file](https://developers.zenodo.org/#add-metadata-to-your-github-repository-release) to have zenodo automatically extract metadata about releases.

### Time Estimate ###
< 1 day

### Sub-Issues ###
Consider breaking the task down into sub-issues.
- [ ] *Add a checkbox for each sub-issue here.*

### Relevant Deadlines ###
None

### Funding Source ###
Multiple keys provided by Tara in email to Julie

### Assignee ###
- [x] Select **engineer(s)** or **no engineer** required
- [ ] Select **scientist(s)** or **no scientist** required

### Labels ###
- [x] Select **component(s)**
- [x] Select **priority**
- [x] Select **requestor(s)**

### Projects and Milestone ###
- [x] Select **Repository** and/or **Organization** level **Project(s)** or add **alert: NEED PROJECT ASSIGNMENT** label
- [x] Select **Milestone** as the next official version or **Future Versions**

## Define Related Issue(s) ##
Consider the impact to the other METplus components.
- [x] [METplus](https://github.com/dtcenter/METplus/issues/new/choose), [MET](https://github.com/dtcenter/MET/issues/new/choose), [METdatadb](https://github.com/dtcenter/METdatadb/issues/new/choose), [METviewer](https://github.com/dtcenter/METviewer/issues/new/choose), [METexpress](https://github.com/dtcenter/METexpress/issues/new/choose), [METcalcpy](https://github.com/dtcenter/METcalcpy/issues/new/choose), [METplotpy](https://github.com/dtcenter/METplotpy/issues/new/choose)

## Task Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [ ] Complete the issue definition above, including the **Time Estimate** and **Funding Source**.
- [ ] Fork this repository or create a branch of **develop**.
Branch name: `feature_<Issue Number>_<Description>`
- [ ] Complete the development and test your changes.
- [ ] Add/update log messages for easier debugging.
- [ ] Add/update unit tests.
- [ ] Add/update documentation.
- [ ] Push local changes to GitHub.
- [ ] Submit a pull request to merge into **develop**.
Pull request: `feature <Issue Number> <Description>`
- [ ] Define the pull request metadata, as permissions allow.
Select: **Reviewer(s)** and **Linked issues**
Select: **Repository** level development cycle **Project** for the next official release
Select: **Milestone** as the next official version
- [ ] Iterate until the reviewer(s) accept and merge your changes.
- [ ] Delete your fork or branch.
- [ ] Close this issue.

### PR DESCRIPTION
**Pull request by jprestop**
## Expected Differences ##

- [x] Do these changes introduce new tools, command line arguments, or configuration file options? **[No]**</br>
If **yes**, please describe:</br>

- [x] Do these changes modify the structure of existing or add new output data types (e.g. statistic line types or NetCDF variables)? **[No]**</br>
If **yes**, please describe:</br>

## Pull Request Testing ##

- [x] Describe testing already performed for these changes:</br>
Used this [JSON Formatter and Validator](https://jsonformatter.curiousconcept.com/#) to ensure formatting and validity.

- [x] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>
Review the files changed.  I don't know if you want to do a test run up to zenodo or wait to see how it works with a release.  I'll leave that up to you.  Please let me know if I need to do anything further.
There are other entries we could add.  See "Deposit metadata" on this [page](https://developers.zenodo.org/#representation).  Some of the entries (e.g. title, publication date) seemed like they would require updating with each release, which I tried to avoid.  But, please let me know if you'd like others added.

- [x] Do these changes include sufficient documentation updates, ensuring that no errors or warnings exist in the build of the documentation? **[Yes]**

- [x] Do these changes include sufficient testing updates? **[I'm not sure, addressed above in the testing for reviewer section]**

- [x] Will this PR result in changes to the test suite? **[No]**</br>
If **yes**, describe the new output and/or changes to the existing output:</br>

- [x] Please complete this pull request review by **[20220908]**.</br>

## Pull Request Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [x] Review the source issue metadata (required labels, projects, and milestone).
- [x] Complete the PR definition above.
- [x] Ensure the PR title matches the feature or bugfix branch name.
- [x] Define the PR metadata, as permissions allow.
Select: **Reviewer(s)**
Select: **Organization** level software support **Project** or **Repository** level development cycle **Project**
Select: **Milestone** as the version that will include these changes
- [x] After submitting the PR, select **Development** with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue and delete your feature or bugfix branch from GitHub.

### CODE REVIEW

File: .zenodo.json [hunk 1]
```diff
@@ -0,0 +1,128 @@
+{
+    "creators": [
+        {
+            "orcid": "0000-0002-7761-4962",
+            "affiliation": "",
+            "name": "Halley Gotway, John"
+        },
+        {
+            "orcid": "",
```
Comment (JohnHalleyGotway) [inline .zenodo.json]:
```suggestion
            "orcid": "0000-0002-4066-0965",
```

File: .zenodo.json [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+            "affiliation": "National Center for Atmospheric Research",
+            "name": "Jensen, Tara"
+        },
+        {
+            "orcid": "",
```
Comment (JohnHalleyGotway) [inline .zenodo.json]:
```suggestion
            "orcid": "0000-0002-9084-9766",
```

File: .zenodo.json [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+            "affiliation": "National Center for Atmospheric Research",
+            "name": "Kalb, Christina"
+        },
+        {
+            "orcid": "",
+            "affiliation": "National Center for Atmospheric Research",
+            "name": "Linden, Seth"
+        },
+        {
+            "orcid": "",
```
Comment (JohnHalleyGotway) [inline .zenodo.json]:
```suggestion
            "orcid": "0000-0001-6266-9090",
```

**Review by JohnHalleyGotway — APPROVED** [review]
FYI, I transferred some OrcID's over from this page:
https://zenodo.org/record/6392131

To fill in some of the missing ones here. That's probably good enough for now, and we can encourage others to sign up for one and add them here in the future.

I approve, after the inclusion of these other OrcID's.

**Comment by jprestop** [comment]
Awesome!  Thank  you!

**Pull request merged**